solr-ingest/src/main/java/edu/unc/lib/dl/data/ingest/solr/filter/SetAccessStatusFilter.java

```diff
@@ -105,13 +108,45 @@ public void setObjectAclFactory(ObjectAclFactory oaf) {
             status.add(FacetConstants.PUBLIC_ACCESS);
         }
 
-        if (objPrincRoles.size() > 0) {
+        if (addPatronSettings(dip.getContentObject())) {
             status.add(FacetConstants.PATRON_SETTINGS);
         }
 
         return status;
     }
 
+    private boolean addPatronSettings(ContentObject contentObj) {
+        List<String> objTypes = contentObj.getTypes();
```

Comment:
you should do `contentObj instanceof AdminUnit` instead of retrieving the types and comparing them